### ISSUE TITLE
Chat: Cyrillic letters for Russian(speaking) users.

### ISSUE THREAD

**Issue by Pingvinszar2** [runelite/runelite#11272]
I would expect this plugin work SIMILAR to chat timestamps. (Overlays, I would guess).

What do I expect from this plugin, and how would this work:

a) RuneLite client automatically would recognize the russian keyboard is active.
**Option 1:** 
Currently, you can't type cyrillic letters in OSRS.
**Step 1:** I would expect a "translator" of some kind, that would convert the Russian Keys into their latin-shortcuts (ie. Я into Ja/Ya, Ю, into Ju/Yu, etc). so the game client can understand the inputs, while your OS Russian Keyboard is active.
**Step 2:** Afterwards, with an overlay, they would be translated again, and this time visualized as Cyrillic letters.

**Option 2:**
b) RuneLite gets a key-combination added, so texts that are meant, would be written in Russian regardless the OS Keyboard layout.
I would expect a key-combination for RuneLite to switch to Russian keyboard (and ability to switch between Mnemonic or Regular).

Avoid conflict with other texts:
I guess, it would be possible to add a pre-condition of some kind, not sure though to be very honest.

**Comment by Nightfirecat** [comment]
I don't think this is a feature which we could reliably maintain. In my opinion you'd be better served finding and using some software for keyboard internationalization or keyboard layout and using that instead.

**Comment by Pingvinszar2** [comment]
> 
> 
> I don't think this is a feature which we could reliably maintain. In my opinion you'd be better served finding and using some software for keyboard internationalization or keyboard layout and using that instead.

I'm using **UK-Based (edited) keyboard** for latin, and **Mnemonic Russian Keyboard Layouts** when I use russian language.

Let me describe you exactly in a different way, what I'm talking about with a certain example.

I will use a text that literally means "I live here." I press down the same keys in the same order.

**When you use an UK Keyboard (or possibly US) it would look like this:**

Ja xivu zdes'.

**But when you switch to a (Mnemonic Russian keyboard) the text will look like this:**

Я живу здедь.

**My problem is the following:**
Although the game CAN recognize the text written with UK keyboard (Ja xivu zdes'). 
The game will not accept the text written with cyrillian letters (Я живу здесь).

I have no problem at all in terms of keyboard layouts or in managing them.

What I want is, an overlay that would REPLACE the (Ja with Я) , (x with ж), (i with и), etc).

I think, that it should be possible with some kind of text-overlay similar to timestamps.
